src/main/scala/tech/cryptonomic/conseil/tezos/ServiceOperations.scala

```diff
@@ -0,0 +1,72 @@
+package tech.cryptonomic.conseil.tezos
+
+import com.typesafe.config.Config
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.collection.JavaConverters._
+
+
+object ServiceOperations {
+
+  final case class Network(name: String, displayName: String, platform: String, network: String)
+
+  final case class Entity(name: String, displayName: String, count: Int, network: String)
+
+  def getNetworks(config: Config): List[Network] = {
+    config.getObject("platforms").asScala.flatMap {
+      case (platform, strippedConf) =>
+        strippedConf.atKey(platform).getObject(platform).asScala.map {
+          case (network, _) =>
+            Network(network, network.capitalize, platform, network)
+        }.toList
+    }.toList
+  }
+
+  def getEntities(network: String)(implicit ec: ExecutionContext): Future[List[Entity]] = {
+    val blocksCountFut = ApiOperations.countBlocks
```

Comment:
I'd suggest to compose the `TezosDB` `DBIO` results and run them all at this level if possible, so slick will take care of combining the queries and get a single Future result back.

WDYT?